### ISSUE TITLE
ReleaseSDK crash (with example link)

### ISSUE THREAD

**Issue by Deviaphan** [ultralight-ux/Ultralight#157]
Example browser crashes on sort of sites. (wiki and sites from src works fine)
As example of crash: https://www.kinopoisk.ru/

Browser builded with MSVC-2019, x64. Project configured with CMake. 

I use True-Debug SDK and True-Debug Browser example (with /MDd switch), to look crash-stack, but it works correct!
I think, some problem with Release version of library.

**Comment by Deviaphan** [comment]
Debug SDK, example with /MDd switch

![Clipboard Image](https://user-images.githubusercontent.com/20601357/65044966-26213180-d988-11e9-8e08-009f74ae27be.png)

Release SDK. "Not responding" and crash

![Clipboard Image (1)](https://user-images.githubusercontent.com/20601357/65045117-6e405400-d988-11e9-824e-6a651ee44fe0.png)

**Comment by adamjs** [comment]
Thanks for this! I had noticed several other JavaScript-heavy websites were also crashing in release but didn't have a reliable method to test.

I was able to investigate and trace the release build (had to force msvc to generate PDB's for optimized build) which identified a race condition in JavaScriptCore's parallel garbage collector marking routine (likely limited to the version of JSC we are currently based on).

I've gone ahead and disabled parallel garbage collection until we can rebase with an updated build-- this seems to have fixed the issue for now. The updates will be in the upcoming 1.1 release.
